gui/src/callbacks.cc

```diff
@@ -443,10 +444,16 @@ void on_scroom_bootstrap(const FileNameMap& newFilenames)
   }
 
 
-  aboutDialogXml = glade_xml_new(xmlFileName.c_str(), "aboutDialog", nullptr);
+  aboutDialogXml = gtk_builder_new();
```

Comment:
We're still leaking this builder, right? (actually two times, due to the duplication ;-))